packages/core/src/interactable-card/InteractableCard.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  forwardRef,
+  MouseEvent,
+  ComponentPropsWithoutRef,
+  SyntheticEvent,
+  useRef,
+} from "react";
+import { clsx } from "clsx";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { capitalize, makePrefixer, useControlled, useForkRef } from "../utils";
+import { useInteractableCard } from "./useInteractableCard";
+import interactableCardCss from "./InteractableCard.css";
+import {
+  InteractableCardValue,
+  useInteractableCardGroup,
+} from "./InteractableCardGroupContext";
+
+const withBaseName = makePrefixer("saltInteractableCard");
+
+export interface InteractableCardProps extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * Accent border position: defaults to "bottom"
+   */
+  accent?: "bottom" | "top" | "left" | "right";
+  /**
+   * **Deprecated:** Use the `accent` prop instead
+   *
+   */
+  accentPlacement?: "bottom" | "top" | "left" | "right";
+  /**
+   * If `true`, the card will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * If `true`, the card will have selected styling.
+   */
+  selected?: boolean;
+  /**
+   * Callback fired when the selection changes.
+   * @param event
+   */
+  onChange?: (event: SyntheticEvent<HTMLDivElement>) => void;
+  /**
+   * Styling variant; defaults to "primary".
+   */
+  variant?: "primary" | "secondary";
+  /**
+   * Value of card (for selectable use case).
```

Comment:
Value can be singular or an array of strings
```suggestion
   * Value of the card(s) (for the selectable use case).
```

Reply:
in the individual card the value is only for one card